<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="apples-MacBook-Air.local" ignored="3" name="Default test(failed)(failed)" tests="2" failures="1" timestamp="2023-01-05T09:03:40 IST" time="19.242" errors="0">
  <testcase name="Test5" time="0.116" classname="TestCases.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FailedTest.Test5(FailedTest.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
  <testcase name="Test3" time="0.323" classname="TestCases.FailedTest"/>
  <testcase name="Test1" time="0.0" classname="TestCases.FailedTest">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.0" classname="TestCases.FailedTest">
    <ignored/>
  </testcase> <!-- Test2 -->
  <testcase name="Test4" time="0.0" classname="TestCases.FailedTest">
    <ignored/>
  </testcase> <!-- Test4 -->
</testsuite> <!-- Default test(failed)(failed) -->
